app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    result = []
+    neg_count = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Instead of returning immediately, append the message to the `result` list: `result.append("All friends should be vaccinated")`. This ensures all friends are checked before deciding the outcome.